From 4e353776f99068ee1c82c4e0768507f0b4fe3022 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Mon, 12 Nov 2012 14:38:39 +0200
Subject: [PATCH 138/227] fmd: re-enable the use of Host Command port

The use of the HC port must be enabled for all platforms.

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
[Kevin: Original patch taken from fsl sdk 1.3.1
QorIQ-SDK-V1.3.1-SOURCE-20121220-yocto.iso.]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../dpa/NetCommSw/src/wrapper/lnxwrp_fm_port.c     |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/lnxwrp_fm_port.c b/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/lnxwrp_fm_port.c
index 08dd422..96d4dd7 100644
--- a/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/lnxwrp_fm_port.c
+++ b/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/lnxwrp_fm_port.c
@@ -751,11 +751,7 @@ static t_Error InitFmPcdDev(t_LnxWrpFmDev *p_LnxWrpFmDev)
 		fmPcdParams.numOfClsPlanEntries = 0;
 		fmPcdParams.partitionId = 0;
 #endif /* CONFIG_MULTI_PARTITION_SUPPORT */
-#if (DPAA_VERSION >= 11)
-		fmPcdParams.useHostCommand = FALSE;
-#else
 		fmPcdParams.useHostCommand = TRUE;
-#endif
 
 		p_LnxWrpFmDev->hc_tx_fq =
 			FqAlloc(p_LnxWrpFmDev,
-- 
1.7.9.7

